src/valkey/valkey.zig

```diff
@@ -5,9 +5,13 @@
 pub const ValkeyContext = @import("./ValkeyContext.zig");
 
 /// Connection flags to track Valkey client state
-pub const ConnectionFlags = packed struct(u8) {
+pub const ConnectionFlags = packed struct(u9) {
```

Comment:
If we decide to keep this packed, I've been told that Zig is inefficient at handling non-power-of-two-sized integers, so it might be better to change the backing integer to `u16`.